src/app/beer_garden/api/stomp/transport.py

```diff
@@ -117,6 +117,8 @@ def on_message(self, headers: dict, message: str):
         Returns:
             None
         """
+        logger.debug(f"Message:\n\tMessage: {message}\n\tHeaders: {headers}")
```

Comment:
This feels like it should be in the other PR for STOMP

Reply:
Ok, I'll move it.